### ISSUE TITLE
opentimestamps.org/internet-archive 404 😐

### ISSUE THREAD

**Issue by marsrobertson** [opentimestamps/opentimestamps.org#44]
* https://en.wikipedia.org/wiki/OpenTimestamps#cite_note-15
* https://bitcoinmagazine.com/articles/opentimestamps-has-timestamped-entire-internet-archive-heres-how/
* https://petertodd.org/2017/carbon-dating-the-internet-archive-with-opentimestamps
* **https://opentimestamps.org/internet-archive/ 404 😐**

![image](https://user-images.githubusercontent.com/44369284/57181962-e56dc380-6e91-11e9-841b-d9e77409e477.png)

**Comment by petertodd** [comment]
My response (so others can see): https://twitter.com/peterktodd/status/1124876271639498752

I do intend to fix this! Just not high on the list. :/

**Comment by petertodd** [comment]
(and if anyone else wants a project to get this up and running, feel free to ask!)

**Comment by moogamouth** [comment]
> (and if anyone else wants a project to get this up and running, feel free to ask!)

@petertodd Yes please, and can you make the backend (including the Internet Archive timestamper itself) open source?

**Comment by petertodd** [comment]
It is open source!

https://github.com/opentimestamps/opentimestamps-server/tree/internetarchive

**Comment by moogamouth** [comment]
> It is open source!
> 
> https://github.com/opentimestamps/opentimestamps-server/tree/internetarchive

Excellent! Is there a publicly accessible backup of the internet archive .ots files? @petertodd

**Comment by petertodd** [comment]
Yes! The entire thing is on the Internet Archive itself: https://archive.org/details/opentimestamp-internetarchive-dataset

**Comment by moogamouth** [comment]
> Yes! The entire thing is on the Internet Archive itself: https://archive.org/details/opentimestamp-internetarchive-dataset

This is from 2018, is there one from 2026? Ideally the timestamper should be running continuously, and upload to the Internet Archive periodically. @petertodd
